airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -45,7 +45,11 @@ public String createTableQuery(final JdbcDatabase database, final String schemaN
   @Override
   public boolean isSchemaExists(final JdbcDatabase database, final String outputSchema) throws Exception {
     try (final Stream<JsonNode> results = database.unsafeQuery(SHOW_SCHEMAS)) {
-      return results.map(schemas -> schemas.get(NAME).asText()).anyMatch(outputSchema::equalsIgnoreCase);
+      if (outputSchema.startsWith("\"") && outputSchema.endsWith("\"")) {
+        return results.map(schemas -> schemas.get(NAME).asText()).anyMatch(outputSchema::equals);
+      } else {
+        return results.map(schemas -> schemas.get(NAME).asText()).anyMatch(outputSchema::equalsIgnoreCase);
+      }
     } catch (Exception e) {
       throw checkForKnownConfigExceptions(e).orElseThrow(() -> e);
     }
```

Comment:
Please double check my merge conflict resolution here, this exception catch was added in `master` so I tacked it on the bottom of the if/else statements added in this PR. I am 99% confident it's fine, but the 1% is because I don't have much experience writing in java.

Reply:
this part looks good